mkdocs.yml

```diff
@@ -213,8 +213,8 @@ markdown_extensions:
     # pymdownx.superfences is deprecated.
     # It must be replaced with pymdownx.tabbed but it requires new CSS
     # TODO with material 5 update
-    - pymdownx.superfences:
-        legacy_tab_classes: true
+    - pymdownx.superfences
+        #legacy_tab_classes: true
```

Comment:
Does this not break the fencing?

Reply:
Commented out per @NicolasMassart to enable building.  Have double checked local publishing and it's OK.

Reply:
Reverted in #368
I was only talking about disabling for local build not for final readthedocs build. Sorry for the misunderstanding.